src/main/resources/application.properties

```diff
@@ -1,13 +1,11 @@
 server.servlet.context-path=/eshop
 server.port=8080
 
-spring.liquibase.enabled=true
-
-spring.liquibase.change-log=classpath:db/changelog/changeLog-master.xml
-
-spring.datasource.url=jdbc:mysql://localhost:3306/eshop
-spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/eshop?createDatabaseIfNotExist=true&allowPublicKeyRetrieval=true&useSSL=false&useUnicode=true&characterEncoding=UTF-8&serverTimezone=UTC
 spring.datasource.username=bestuser
 spring.datasource.password=bestuser
 
-spring.h2.console.enabled=true
+spring.jpa.hibernate.ddl-auto=none
+
+spring.jpa.show-sql=true
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
```

Comment:
If you use MySQL 8, then change dialect here to `org.hibernate.dialect.MySQL8Dialect`